generators/server/templates/src/main/java/package/config/_DefaultProfileUtil.java

```diff
@@ -20,14 +18,7 @@
      * @param app the spring application
      */
     public static void addDefaultProfile(SpringApplication app) {
-        Map<String, Object> defProperties =  new HashMap<>();
-        /*
-        * The default profile to use when no other profiles are defined
-        * This cannot be set in the <code>application.yml</code> file.
-        * See https://github.com/spring-projects/spring-boot/issues/1219
-        */
-        defProperties.put(SPRING_PROFILE_DEFAULT, Constants.SPRING_PROFILE_DEVELOPMENT);
-        app.setDefaultProperties(defProperties);
+        app.setAdditionalProfiles(Constants.SPRING_PROFILE_DEVELOPMENT, Constants.SPRING_PROFILE_SWAGGER);
```

Comment:
So the error logs come from mysql:5.7.15 -> I made a PR to revert back to 5.7.13
But the application manages to start because mixing of profile dev+prod